cif_core.dic

```diff
@@ -27613,11 +27613,13 @@ save_function.atomtype
 
        m = Len(s)
        n = 1
-       If (m > 1 and s[1] not in '0123456789') n = 2
-       If (m > 2 and s[2]     in '+-'        ) n = 3
-       If (m > 3 and s[3]     in '+-'        ) n = 4
+       f = ""
+       if (m > 0) f += Upper(s[0])
+       If (m > 1 and s[1] not in '0123456789') f += s[1]
```

Comment:
Maybe s[1] should be lowercased in case we have something like "FE+2"?

Reply:
Good idea